website/source/docs/providers/azure/index.html.markdown

```diff
@@ -33,7 +33,7 @@ resource "azure_instance" "web" {
 
 The following arguments are supported:
 
-* `settings_file` - (Optional) The path to a publish settings file used to
+* `settings_file` - (Required) Contents of a valid `publishsettings` file, used to
```

Comment:
Do we know that this is required? Why was it set as optional in the first place? Maybe there are weird alternative auth mechanisms like instance roles in AWS?